src/Components/App/App.js

```diff
@@ -115,7 +122,7 @@ const App = () => {
       />
       <Route path="/vendor-view" element={
         <div>
-          <VendorView toggleLive={sendData} vendor={vendors.find(v => v.id == 1)} currentVendor={currentVendor}/>
+          <VendorView toggleLive={sendData} getAddress={sendAddress} vendor={vendors.find(v => v.id == 9)} currentVendor={currentVendor}/>
```

Comment:
You'll have to walk me through changing this from an id of 1 to an id of 9-- not sure I get it just looking at the code. Thanks in advance!

Reply:
There is nothing special there. It was to change which  vendor is hardcoded to the vendor view. This way it is Taj Palace, the one that by default is rendered at the top of the list on the main page.